Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1765,6 +1766,20 @@ def lhegensim(fragment,howMuch):
         upgradeStepDict['HARVESTFull_trackingOnlyPU'][k]=merge([PUDataSets[k2],upgradeStepDict['HARVESTFull_trackingOnly'][k]])
         upgradeStepDict['HARVESTFullGlobalPU'][k]=merge([PUDataSets[k2],upgradeStepDict['HARVESTFullGlobal'][k]])
 
+
+
+    upgradeStepDict['ALCAFull'][k] = {'-s':'ALCA:TkAlMuonIsolated+TkAlMinBias+MuAlOverlaps+EcalESAlign',
```

Comment:
@bsunanda @kpedro88 anything Hcal related you might want to be included in here?